README.md

```diff
@@ -1,302 +1,75 @@
-# Table of Contents #
+# AutoSpotting #
```

Comment:
I usually like to have LICENSE at the end of the README, but there is already a LICENSE file, so I guess it's not required.